Weather/Weather/Model/ForecastWeatherHourtly.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  ForecastWeatherHourtlyModel.swift
+//  Weather
+//
+//  Created by Sergii Lisnyi on 5/18/18.
+//  Copyright © 2018 Sergii Lisnyi. All rights reserved.
+//
+
+import Foundation
+
+struct ForecastWeatherHourly {
+    var city: String?
+    var tempCurrent: String?
```

Comment:
I am not sure that any value allowed to be optional here.
`arrayTempHourly`and `arrayTimeHourly` entries should be tuple of time+temperature, I think